### DISCUSSION TITLE
code-server for raspbbery pi

### DISCUSSION THREAD

**Discussion by Slodziak190q** [coder/code-server#4473] [Ideas]
Create a new code-server version without argon2 that will work like latest version of code-server

**Comment by lg-nineoneninethreenine** [comment]
Is this a feature request?

**Reply by lg-nineoneninethreenine** [reply]
For now, 3.10.2 works best since Argon2 was merged on 3.11.0, but removing Argon2 is bad since it is used for hashing passwords.

**Comment by jsjoeio** [comment]
Follow this issue: https://github.com/cdr/code-server/issues/4422 - we're aware and hope to find a solid fix
